src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -280,6 +281,37 @@ public static string CleanFolderName(string name)
             return name.Trim(' ', '.');
         }
 
+        public bool RequiresEpisodeTitle(Series series, List<Episode> episodes)
+        {
+            var namingConfig = _namingConfigService.GetConfig();
+            var pattern = namingConfig.StandardEpisodeFormat;
+
+            if (series.SeriesType == SeriesTypes.Daily)
+            {
+                pattern = namingConfig.DailyEpisodeFormat;
+            }
+
+            if (series.SeriesType == SeriesTypes.Anime && episodes.All(e => e.AbsoluteEpisodeNumber.HasValue))
```

Comment:
Why check if the episode has absolute numbers filled in? The renamer itself doesn't. Afaik it won't even import if there's no abs nr.

Reply:
If the absolute episode numbers are not set the standard format is used instead.

https://github.com/Sonarr/Sonarr/blob/968d8159a6c8bc3f9d4168a8940a745c240d1b25/src/NzbDrone.Core/Organizer/FileNameBuilder.cs#L121-L124

Whether that's what we should do or not is questionable, with another PR discussing it here: https://github.com/Sonarr/Sonarr/pull/2283

Reply:
Understood, we'll revisit that later then.